ibis/tests/benchmarks/test_benchmarks.py

```diff
@@ -691,7 +695,10 @@ def test_compile_with_drops(
     except (AttributeError, ImportError) as e:
         pytest.skip(str(e))
     else:
-        benchmark(mod.compile, expr)
+        try:
+            benchmark(mod.compile, expr)
+        except sa.exc.NoSuchModuleError as e:
+            pytest.skip(str(e))
```

Comment:
When not all the backend dependencies are installed.

Reply:
To be more precise, we now attempt to load the sqlalchemy dialect when compiling, to properly handle type translation (part of the purpose of this PR). When a dialect fails to load then this error is raised.

We still do a sanity check on the benchmark so instead of failing we skip the test.